sql/core/src/main/scala/org/apache/spark/sql/execution/datasources/jdbc/JdbcUtils.scala

```diff
@@ -114,22 +115,19 @@ object JdbcUtils extends Logging with SQLConfHelper {
       isCaseSensitive: Boolean,
       dialect: JdbcDialect): String = {
```

Comment:
note: `getInsertStatement` is not a public API but we don't stop people from calling `JdbcUtils` as there is no `private[spark]`. I'm fine keeping it to avoid troubles for third-party Spark vendors.